pkg/cmd/cli/describe/printer.go

```diff
@@ -159,6 +166,30 @@ func PrintTemplateParameters(params []templateapi.Parameter, output io.Writer) e
 	return nil
 }
 
+// Look-up for resource short forms by value
+func resourceShortFormFor(resource string) (string, bool) {
```

Comment:
The godoc convention is to start a function documentation with its name:

```go
// resourceShortFormFor looks up for a short form of resource names.
func resourceShortFormFor(resource string) (string, bool) {
```

https://golang.org/doc/effective_go.html#commentary